packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageModule.kt

```diff
@@ -171,7 +171,9 @@ class ExpoImageModule : Module() {
       }
 
       OnViewDestroys { view: ExpoImageViewWrapper ->
-        view.onViewDestroys()
+        view.doOnDetach {
```

Comment:
You forgot to import this function.
Please add `import androidx.core.view.doOnDetach` on top of the file.